### DISCUSSION TITLE
Automatic sending of messages when the macro is triggered

### DISCUSSION THREAD

**Discussion by DarkNirDanil** [nlef/moonraker-telegram-bot#232] [Ideas]
Hello.
Tell me if it is realistic to implement the function of automatically sending messages from the printer in the Telegram Bot when the macro is triggered.
For example: I have a filament motion sensor, and if the filament is over or stuck, the printer goes into pause. It would be great to get a notification from the printer so that I know that I need to check the filament, and not check it myself.
----------------------------------------------------------
Здравствуйте.
Подскажите, реально ли реализовать функцию по автоматической отправки сообщений от принтера в Телеграм Бот при срабатывании макроса.
Для примера: у меня стоит датчик движения филамента, и если филамент закончился или застрял принтер уходит в паузу. Было бы отлично получить от принтера уведомление, чтобы я знал, что нужно проверить филамент, а не проверял бы его сам.

**Comment by aka13-404** [comment]
Yes, this is exaclty what the bot is for. 
See https://github.com/nlef/moonraker-telegram-bot/wiki/Interacting-with-klipper#send-custom-notifications-to-the-bot
And 
https://github.com/nlef/moonraker-telegram-bot/wiki/Ideas-for-macros 
as examples
